pkg/patterns/declarative/pkg/applier/direct_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !without_kubectl_deps
```

Comment:
Maybe we do this by applier, because there are two "dependencies" on kubectl - there is execing kubectl, and there is calling into the kubectl code.

So maybe we have two tags: `without_kubectl_applier` (or `without_exec_applier`) and `without_direct_applier` ?